### ISSUE TITLE
wrong sed expression in provision shell script used by Vagrantfile  

### ISSUE THREAD

**Issue by rafaeltuelho** [fabric8io/fabric8-installer#86]
I've noticed a wrong `sed` expression used at [Vagrantfile line 69](https://github.com/fabric8io/fabric8-installer/blob/master/vagrant/openshift/Vagrantfile#L69)

This `sed` produces the following line after executed:

```
podManifestConfig:\n  path: /var/lib/openshift/openshift.local.manifests\n  fileCheckIntervalSeconds: 10
```

It should produce

```
podManifestConfig:
  path: /var/lib/openshift/openshift.local.manifests
  fileCheckIntervalSeconds: 10
```

You are escaping the newline char (`\\n`). This produce a malformed `YAML` config file for the node. This malformed YAML crashes the provision execution.

**Comment by jimmidyson** [comment]
This hasn't changed in nearly a year & has been widely used. Are you trying to use this outside of the Vagrantfile? I can't quite remember but the double escaping might be due to the fact that it's inside a ruby string (Vagrantfile is just a ruby source file really) so if you're trying to reuse the sed elsewhere then you will need to remove the double escaping.

**Comment by rafaeltuelho** [comment]
Hum!

You are right! I had to execute the script outside the Vagrantfile. I am running it in my box (a RHEL 7 with KVM/libvirt). For some reason (in my box) the provision step does note executes during `vagrant up` :-\

After a while I discovered that I need to issue a `vagrant provision` after the `vagrant up` .

Now it seems to be working on a `RHEL7` box with `KVM/libvirt`.

BTW, It appears the majority of fabric8 fellows are using a Mac or a M$ Windows box :-)
